packages/driver/cypress/plugins/server.js

```diff
@@ -296,17 +297,20 @@ const createApp = (port) => {
   })
 
   app.get('/test-request-credentials', (req, res) => {
+    const origin = cors.getOrigin(req['headers']['referer'])
```

Comment:
since many of the requests are now same origin, we don't get the `origin` header, so we grab it from the referrer for the cookie_behavior tests